terraform/gcp/k8s/site-config.yml

```diff
@@ -0,0 +1,158 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: d2p-site-configmap
+data:
```

Comment:
Do I remember right that there's only one proxy per that serves both admin and participant?